### PR TITLE
use callback-hook package

### PR DESCRIPTION
**Pull request by brucejo75**
In response to https://github.com/meteor/meteor/pull/11607#issuecomment-907904575

For the 2 callbacks I introduced I have updated to use the meteor `callback-hook` package.

### CODE REVIEW

**Comment by brucejo75** [comment]
Working on the History.md now.

File: packages/webapp/webapp_server.js [hunk 1]
```diff
@@ -435,7 +433,7 @@ WebAppInternals.generateBoilerplateInstance = function (arch,
     ...__meteor_runtime_config__,
     ...(additionalOptions.runtimeConfigOverrides || {})
   };
-  runtimeConfig.updateHooks.forEach((cb) => {
+  runtimeConfig.updateHooks.each((cb) => {
```
Comment (StorytellerCZ) [inline packages/webapp/webapp_server.js:436]: You can use here `forEach`, it has been added in `callback-hook@1.4.0` which is available on the release branch.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

**Comment by brucejo75** [comment]
@StorytellerCZ, I am working on the api box stuff.  If you can give me another day, I will fold it into this PR (need to add into the code).

**Review by StorytellerCZ — APPROVED** [review]

**Comment by filipenevola** [comment]
@brucejo75 last changes like this are dangerous, make sure you test these changes properly 😉

**Comment by brucejo75** [comment]
@StorytellerCZ, @filipenevola,

I have documented the new methods in `WebApp` via jsdoc & `apibox`.  While I was at it I moved `connectHandlers` documentation to `apibox` as well.

**Note**: this documentation depends on https://github.com/meteor/docs/pull/753 & uses a new tag `@isprototype` to suppress the `importName` & `filepath` from the `api-heading` when documenting callback functions.

**Pull request merged**